src/sinks/splunk_hec/logs/config.rs

```diff
@@ -40,19 +40,22 @@ pub struct HecLogsSinkConfig {
     pub default_token: SensitiveString,
 
     /// The base URL of the Splunk instance.
+    #[configurable(validation(format = "uri"))]
     pub endpoint: String,
```

Comment:
I believe there are other places this URI validation applies to that aren't covered by this PR (humio sinks, http_client for example). 
It this PR meant to catch them all ?

Reply:
This PR is one of many, so more areas will be annotated, etc, yes.